### PR TITLE
[Tabs][Newdocandupdatecontent][1196825]

### PR DESCRIPTION
**Pull request by KirtiPereira**

### CODE REVIEW

**Comment by KirtiPereira** [comment]
@lokimeyburg - please review the PR and provide your comments / approval.
Thank you.

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+  "version": "0.0.1",
+  "packageName": "acprototype",
+  "developer": {
+    "name": "proto",
+    "websiteUrl": "https://acprototype.azurewebsites.net",
+    "privacyUrl": "https://acprototype.azurewebsites.net/privacy.html",
+    "termsOfUseUrl": "https://acprototype.azurewebsites.net/tou.html"
+  },
+  "name": {
+    "short": "Workday",
```
Comment (Anoop-D) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:48]: We should probably remove references to Workday in this sample manifest

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    "value: {
+        "tabContext": {
+            "tabEntityId": "{tab_entity_id}"
+        },
+        "context": {
+            "theme": "default"
+            }
+    },
+    "conversation": {
+        "id": "tab:hash(tabId_appId_userId)"
```
Comment (Anoop-D) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:119]: I don't think we want to share how this conversation id is generated with the public. It's probably best to keep this obfuscated, something like tab:conversationId? or just "generated conversation id"

Comment (KirtiPereira) [reply msteams-platform/tabs/how-to/build-adaptive-card-tabs.md]: updated to "generated conversation id"

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md (hunk 1 above)
Comment (KirtiPereira) [reply msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:48]: @lokimeyburg - to share the update.

**Comment by KirtiPereira** [comment]
@lokimeyburg - please share the inputs.
Thanks.

**Review by Anoop-D — DISMISSED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approve

**Review by v-abirade — APPROVED** [review]
Approved

**Review by vikramjit-singh-chug — APPROVED** [review]

**Pull request merged**